community.md

```diff
@@ -143,6 +145,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 
 * [Blazor First Look on LinkedIn Learning](https://www.linkedin.com/learning/blazor-first-look)
 * [Free Blazor Training Course](https://www.devexpress.com/support/training/blazor/)
+* [Blazor StateHasChanged: The weekly community live show](https://www.youtube.com/edwardcharbeneau)
```

Comment:
This isn't really a course. It could go under Other, or maybe we should a new section for Streams / Podcasts?

Reply:
:heart: "Streams and Podcasts"

I'll move it to that. Stand-by.